buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -54,6 +62,45 @@ public void apply(Project project) {
                 }
               });
     }
+    if (System.getenv().containsKey("METALAVA_BINARY_PATH")) {
+      String metalavaBinaryPath = System.getenv("METALAVA_BINARY_PATH");
+      AndroidSourceSet mainSourceSet = android.getSourceSets().getByName("main");
+      File apiTxtFile = new File(project.getProjectDir() + "/api.txt");
+      File baseLineFile = new File(project.getProjectDir() + "/baseline.txt");
```

Comment:
What's the purpose of baseline.txt? it seems to be empty, also why is it's version declared as 1.0, while api.txt is declared as 2.0?

Reply:
Baseline files are certain files which allows metalava to ignore the warnings present in these files. There is a system property updateBaseline which is added which allows us to control updation of the baseline file.

Reply:
These files are empty for some sdks whereas its not for firestore, common and other sdks